src/app/pages/plugins/plugin-add/plugin-add.component.ts

```diff
@@ -339,7 +339,7 @@ export class PluginAddComponent implements OnInit {
             }
           );
         } else {
-          this.snackBar.open(T("Plugin sucessfully installed"), T("Close"), { duration: 5000 });
+          this.snackBar.open(T("Plugin successfully installed"), T("Close"), { duration: 5000 });
```

Comment:
Do we need "successfully" at all?  I mean, if it installed, it was successful.

Reply:
fixed